app.rb

```diff
@@ -4,13 +4,40 @@
 require 'byebug'
 
 require_relative 'publisher.rb'
+require_relative 'subscriber.rb'
+
+class App
```

Comment:
No need for class here, there will always be a single instance of the application being run.